README.md

```diff
@@ -12,7 +12,10 @@ A mongodb database named lizardboard must be created prior to starting the appli
 - brew install yarn
```

Comment:
Please revert changes in this file - these are recommendations, not instructions for setting up the application.  See the floworky README for guidance adding instructions for setting up .env